css/style.css

```
@@ -16,4 +16,129 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* This is Header (Navigation & Hero )*/
+
+/* Navigation */
+nav{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:140px;
+    width: 80%;
+    margin: auto;
+    font-size: 2.5rem;
+    }
+
+a{
+    height: 60px;
+    margin-left: 80px;
+    text-decoration: none;
+    color: rgb(65, 63, 63);
+    }
+
+a:hover{
+    color: rgb(250, 115, 4);
+    }
+
+.logo{
+    width:50px;
+    height:auto;
+    margin-right: auto;
+    margin-left: 0;
+    }
+
+.logo img{
+    width: 100%;
+    height: auto;
+    }
+
+ /* Hero style */
+.hero-image{
+    background-image: url(../img/first-background.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 900px;
+    position:relative;
+}
+
+.hero-text{
+    text-align:center;
+    position: absolute;
+    color: white;
+    top:20%;
+    left:40%;
+}
+
+.hero-text h1{
+    font-size: 5rem;
+}
+
+.hero-text p{
+    font-size: 3rem;
+}
+
+button {
+    background-color: #f2750e; /* Orange */
+    border: none;
+    border-radius: 5px;
+    color: white;
+    font-size: 2rem;
+    padding: 20px 30px;
```

```suggestion
    padding: 1.25rem 2rem;
```
We should use relative units like rem instead of absolute units like px where possible. Same goes for the rest of this file